### ISSUE TITLE
make transmit_rps.sh chattier so we have more information to use in troubleshooting

### ISSUE THREAD

**Issue by jasonfleming** [StormSurgeLive/asgs#622]

**Comment by jasonfleming** [comment]
Adding commentary from Slack so we have a record of it:
Hello All, I have been testing the RENCI DB run.properties notifier this morning with the `EC95d_al072021_jgf` instance of ASGS on Stampede2. I tried it as-is, with the master of a couple days ago, and it created a `rps.transmit.succeeded` file for advisory 16. However, I have to wonder whether it actually did succeed. Can anyone with access to the DB check this?
My next step was to update to the latest master and try again. By this time, Grace advisory 17 had come out, so it ran that. This time, in the `nhcConsensus` directory, the `transmit_rps.sh file` wrote a `rps.transmit.failed` file. Unfortunately, it does not say why it failed. @BrianOBlanton can you make a branch off the latest master for holding changes related to this so we can determine root cause?
FYI, it looks like configuration is in good shape:
```
asgs (EC95d_al072021_jgf)> pwd
/scratch/00976/jgflemin/asgs458640/17/nhcConsensus
asgs (EC95d_al072021_jgf)> grep -i rmqmessaging run.properties
monitoring.rmqmessaging.enable : on
monitoring.rmqmessaging.transmit : on
monitoring.rmqmessaging.script : /work2/00976/jgflemin/stampede2/asgs/monitoring/asgs-msgr.py
monitoring.rmqmessaging.scriptrp : /work2/00976/jgflemin/stampede2/asgs/monitoring/rp2json.py
monitoring.rmqmessaging.ncohome : /work/00976/jgflemin/stampede2/local
monitoring.rmqmessaging.locationname : TACC
monitoring.rmqmessaging.clustername : Stampede2
asgs (EC95d_al072021_jgf)> grep -i post.executable.postprocess run.properties
post.executable.postprocess : (  includeWind10m.sh createOPeNDAPFileList.sh opendap_post.sh transmit_rps.sh )
asgs (EC95d_al072021_jgf)> cat rps.transmit.failed
Tue Aug 17 10:04:00 CDT 2021
```
Oh I also saw the earlier ASGS instance (with the older master) show up in the ASGS Monitor, but not once I updated to the latest master.

**Comment by BrianOBlanton** [comment]
Happy to do this, as this is how I found/fixed the problem in the first place, or so I thought.  At the moment, I can't get ASGS to install completely, through init-asgs.

**Comment by jasonfleming** [comment]
Hey @BrianOBlanton I just tested this on Stampede2 and while it does not seem to have worked, we do have more info to go on. Here are the relevant messages I saw on the console:
```
RMQ : EC95d_al072021_jgf :         21 : INFO : FORE : 2021-Aug-18-T15:47:16 : WAIT :   0.0 : asgs_main.sh>nhcConsensus : transmit_rps.sh 07 2021 21 stampede2.tacc.utexas.edu nhcConsensus 2021071500    2980800.00000000      ec_95d.grd /work2/00976/jgflemin/stampede2/asgs/output
Found run.properties file
/work2/00976/jgflemin/stampede2/asgs/output/transmit_rps.sh: line 31: allMessage: command not found
/work2/00976/jgflemin/stampede2/asgs/output/transmit_rps.sh: line 83: return: can only `return' from a function or sourced script
```
and here are the messages I saw in the `$SYSLOG`:
```
[2021-Aug-18-T10:47:16-0500] INFO: nowcast: asgs_main.sh: Executing POSTPROCESS hook /work2/00976/jgflemin/stampede2/asgs/output/transmit_rps.sh.
Traceback (most recent call last):
  File "/work2/00976/jgflemin/stampede2/asgs/monitoring/rp2json.py", line 80, in <module>
    sys.exit(main(args))
  File "/work2/00976/jgflemin/stampede2/asgs/monitoring/rp2json.py", line 52, in main
    rp_dict = ReadRPToDict(input_filename, uid, instance_name, physical_location)
  File "/work2/00976/jgflemin/stampede2/asgs/monitoring/rp2json.py", line 15, in ReadRPToDict
    (key, val) = line.split(' : ')
ValueError: need more than 1 value to unpack
[2021-Aug-18-T10:47:18-0500] INFO: /work2/00976/jgflemin/stampede2/asgs/monitoring/rp2json.py
```

**Comment by jasonfleming** [comment]
The run.properties file that it was trying to parse can be found here: http://adcircvis.tacc.utexas.edu:8080/thredds/catalog/asgs/2021/al07/21/ec95d/stampede2.tacc.utexas.edu/EC95d_al072021_jgf/nhcConsensus/catalog.html

**Comment by jasonfleming** [comment]
Ok I just tested this again on Stampede2, and I can't confirm that it worked b/c the associated entry on the ASGS Monitor disappeared as soon as the forecast was complete, but I don't see any error messages on the console:
```
RMQ : EC95d_al072021_jgf :         21 : INFO : FORE : 2021-Aug-18-T18:37:12 : WAIT :   0.0 : asgs_main.sh>nhcConsensus : transmit_rps.sh 07 2021 21 stampede2.tacc.utexas.edu nhcConsensus 2021071500    3002400.00000000      ec_95d.grd /work2/00976/jgflemin/stampede2/asgs/output
```
or in the log file:
```
[2021-Aug-18-T13:37:12-0500] INFO: nowcast: asgs_main.sh: Executing POSTPROCESS hook /work2/00976/jgflemin/stampede2/asgs/output/transmit_rps.sh.
[2021-Aug-18-T13:37:14-0500] INFO: /work2/00976/jgflemin/stampede2/asgs/monitoring/rp2json.py
```

**Comment by BrianOBlanton** [comment]
I'm not sure why the entry on the monitor would disappear.  Did you kill the instance after the test forecast completed?

**Comment by BrianOBlanton** [comment]
Success.

<img width="1022" alt="Screen Shot 2021-08-18 at 3 11 16 PM" src="https://user-images.githubusercontent.com/2234435/129957978-b1317980-797d-4d3c-a2c4-35d38a4bdb0e.png">

**Comment by jasonfleming** [comment]
Ok that is good to hear. I didn't stop the instance, it is still running. I see it has reappeared on the monitor. I will continue to watch it and see what happens for advisory 22.

**Comment by jasonfleming** [comment]
Ok @BrianOBlanton let's observe this on the next advisory for Grace, and if no issues are detected, can you make a PR for this and assign @wwlwpd and me as reviewers?

**Comment by wwlwpd** [comment]
Merged #623, does that fully satisfy this Issue?

**Comment by jasonfleming** [comment]
Yes, sorry for the delay. Thank you for your time and attention on this.
